Chinese/斗战胜佛/斗战胜佛.md

```diff
@@ -1 +1,3 @@
-孙悟空取经回来后，就变成了斗战胜佛！他从一个叛逆者，历经千辛万苦终于成了一个体制内的人，可是沿途的妖怪那么轻松就成仙成佛了，这个世界跟对人十分重要啊！
\ No newline at end of file
+孙悟空取经回来后，就变成了斗战胜佛！他从一个叛逆者，历经千辛万苦终于成了一个体制内的人，可是沿途的妖怪那么轻松就成仙成佛了，这个世界跟对人十分重要啊！
+
+但是其实在西天取经的过程中，原来总希望成仙成佛的孙悟空也慢慢的体会到了，仕途之路并不适合他内心那颗“世界因你而不同”的灵魂。所以回到花果山之后，他决定转世投胎。你看见的李开复其实就是花果山的那只猴子，所以这个世界有梦想还是很重要的，你看癌症都挺过来！
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.